jina/executors/compound.py

```diff
@@ -231,8 +231,8 @@ def save(self, filename: str = None) -> bool:
         """
 
         for c in self.components:
-            c.save()
-        super().save()  # do i really need to save the compound executor itself
+            c.save(filename=filename)
```

Comment:
It seems weird to store the executors formingn the compound executor and the compound executor itself in the same file